src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,66 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final String DEFAULT_COLOR = "white";
+    private static final int RADIUS_DEFAULT = 10;
+    private static final int MAX_RANDOM = 20;
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getDefaultFigure() {
+        return new Circle(RADIUS_DEFAULT, DEFAULT_COLOR);
+    }
+
+    public Figure getRandomFigure() {
+        int randomFigures;
+        randomFigures = random.nextInt(5) + 1;
+        switch (randomFigures) {
+            case 1: {
```

Comment:
```suggestion
            case 0: {
```
it's okay to start counting from 0 in programming, get used to it:)